src/pages/LoanFinding/LoanFinding.vue

```diff
@@ -324,9 +318,33 @@ export default {
 			);
 			this.enableRelendingExp = version === 'b';
 		}
+
+		const recommendedArray = [
+			...cachedRecommendedLoans,
+			{ id: 0 }, { id: 0 },
+			{ id: 0 }, { id: 0 },
+			{ id: 0 }, { id: 0 },
+			{ id: 0 }, { id: 0 },
+		];
+
+		let relendingArray = [
+			...cachedRecommendedLoans,
+		];
+
+		/* eslint-disable max-len */
+		if (this.enableFiveDollarsNotes) {
+			if ((userBalance > 0 && userBalance < 10) || (userBalance > 20 && userBalance < 50)) relendingArray = relendingArray.slice(0, 2);
+			if ((userBalance > 10 && userBalance < 15) || (userBalance > 50 && userBalance < 75)) relendingArray = relendingArray.slice(0, 3);
+		} else {
+			if (userBalance > 0 && userBalance < 50) relendingArray = relendingArray.slice(0, 2);
+			if (userBalance > 50 && userBalance < 75) relendingArray = relendingArray.slice(0, 3);
+		}
+
+		this.userBalance = userBalance;
+		this.recommendedLoans = this.enableRelendingExp ? relendingArray : recommendedArray;
```

Comment:
I would rename `recommendedLoans` to something more generic, like `firstRowLoans` so we don't confuse the two uses.